tests/integration/test_clients.py

```diff
@@ -0,0 +1,62 @@
+import pathlib
```

Comment:
None of your tests are checking for content. How do you know the messages are consumed/produced correctly?